pkgs/top-level/perl-packages.nix

```diff
@@ -7018,9 +7400,9 @@ let
     buildInputs = [ DBIxClass TestSimple13 ];
     propagatedBuildInputs = [ DBDSQLite SQLTranslator ];
     meta = {
-      homepage = "https://metacpan.org/pod/DBICx::TestDatabase";
-      description = "Create a temporary database from a DBIx::Class::Schema";
+      description = "create a temporary database from a DBIx::Class::Schema";
       license = with lib.licenses; [ artistic1 gpl1Plus ];
+      homepage = "https://metacpan.org/pod/DBICx::TestDatabase";
```

Comment:
those "homepages" are to be removed
they are ignored on CPAN importing, but  it is kept because it was so in nixpkgs

Ok, postponed to the next run